microsoft-edge/devtools-guide-chromium/whats-new/2022/02/devtools.md

```diff
@@ -0,0 +1,167 @@
+
+In 2021, developer tooling for Microsoft Edge has taken a massive leap forward.  Visual Studio Code, DevTools, and the browser now provide several integrated workflows for JavaScript debugging, mirroring CSS changes, and screencasting from the browser.  Between Focus Mode, support for Visual Studio Code themes, and further customization options, the Microsoft Edge team is working to make the DevTools easier and simpler to use.
+
+Catch up on all of this news and more in our year-in-review blog post, [Looking back at Microsoft Edge for developers in 2021](https://blogs.windows.com/msedgedev/2022/01/19/looking-back-at-microsoft-edge-for-developers-in-2021/).
+
+[![Blog post: Looking back at Microsoft Edge for developers in 2021.](../../media/2022/02/blog-post-edge-devs-2021.png)](https://blogs.windows.com/msedgedev/2022/01/19/looking-back-at-microsoft-edge-for-developers-in-2021/)
+
+
+<!-- ====================================================================== -->
+## Automate IE mode with Internet Explorer Driver
+
+<!-- careful changing h2 wording: Welcome tool potentially links to it -->
+
+<!-- Title: New support for automating IE mode -->
+<!-- Subtitle: Test your legacy websites and apps by automating IE mode with Internet Explorer Driver. -->
+
+Starting June 15, 2022, Internet Explorer 11 will no longer be supported on certain versions of Windows 10. IE mode is a feature in Microsoft Edge for organizations that still need Internet Explorer 11 for business-critical legacy websites or apps. You can now automate IE mode with Selenium 4 and Internet Explorer Driver. To get started, see [Use Internet Explorer Driver to automate IE mode in Microsoft Edge](../../../../webdriver-chromium/ie-mode.md).
+
+
+<!-- ====================================================================== -->
+## Emulate forced-colors mode
+
+<!-- careful changing h2 wording: Welcome tool potentially links to it -->
+
+<!-- Title: Emulation of forced colors in the Rendering tool -->
+<!-- Subtitle: You can now do a spot check of what your product will look like on a device running in forced-colors mode, without having to change your operating system settings. -->
+
+In addition to emulating dark, light, and print mode for the current webpage, now you can also see what your webpage looks like for users who have forced-colors mode enabled.  Forced-colors mode is an accessibility feature of the operating system.  It forces colors on the webpage to be reduced to a limited color palette, such as Windows High Contrast mode.  Emulation of forced-colors mode allows you to do a spot check without having to change your own system's settings.
+
+To turn on forced-colors emulation, in DevTools, open the **Rendering** tool, and then in the **Emulate CSS media feature forced-colors** dropdown list, select **forced-colors: active**.
+
+![The 'Emulate CSS media feature forced-colors' dropdown list in the Rendering tool.](../../media/2022/02/emulate-forced-colors.png)
```

Comment:
The main concern here is that this screenshot shows where to change this but doesn't give the wow factor of the functionality. Can we do a second screenshot with emulated forced colours mode to show people what it does?